src/edfi-performance-test/edfi_performance_test/api/client/ed_fi_api_client.py

```diff
@@ -78,14 +78,13 @@ def create_with_dependencies(self, **kwargs):
     factory: Any = None
     endpoint: str = ""
     dependencies: Dict = {}
-    token: str = ""
 
     def __init__(self, client: HttpSession, token: str = "", api_prefix: str = ""):
         super(EdFiAPIClient, self).__init__(
             client, token, api_prefix, endpoint=self.endpoint
         )
 
-        EdFiAPIClient.token = token
+        EdFiAPIClient.token = self.token
```

Comment:
When the `token` argument is blank, the `__init__` method above logs into the API and sets the _instance_ variable version of `token` - in other words, it sets `self.token`. Thus changing so that the instance values is copied into the static value after a successful login.